pages/en/developer/assemblyscript-api.mdx

```diff
@@ -188,7 +188,7 @@ The `TypedMap` class has the following API:
 import { Bytes } from '@graphprotocol/graph-ts'
 ```
 
-`Bytes` is used to represent arbitrary-length arrays of bytes. This includes Ethereum values of type `bytes`, `bytes32` etc.
+`Bytes` are used to represent arbitrary-length arrays of bytes. This includes Ethereum values of type `bytes`, `bytes32`, etc.
```

Comment:
I think this was in the sense "[The class] `Bytes` is used to represent..."

So, I would leave it as is:
```suggestion
`Bytes` is used to represent arbitrary-length arrays of bytes. This includes Ethereum values of type `bytes`, `bytes32`, etc.
```